spec/views/startup/index.html.erb_spec.rb

```diff
@@ -0,0 +1,5 @@
+require 'rails_helper'
+
+RSpec.describe "startup/index.html.erb", type: :view do
+  pending "add some examples to (or delete) #{__FILE__}"
```

Comment:
delete this whole file, we dont do view tests.
These are the downside of the generators since they sometimes give us some junk.